src/main/resources/static/openapi/EducationAndWorkPlanAPI.yml

```diff
@@ -987,10 +987,33 @@ components:
           items:
             $ref: '#/components/schemas/WorkExperience'
         workInterests:
-          $ref: '#/components/schemas/WorkInterests'
+          $ref: '#/components/schemas/CreateWorkInterestsRequest'
       required:
         - hasWorkedBefore
 
+    CreateWorkInterestsRequest:
```

Comment:
We need a separate WorkInterests object for the request, due to the lack of an `id` (i.e. external reference).

Note that unlike the other 1st class citizen objects in the API, `WorkInterests` does not have `modifiedBy` or `modifiedDateTime` fields, which means that both the PUT (update) request object and the GET response object are identical. As a result, `WorkInterests` is used by both the PUT and GET endpoints, but the POST (create) endpoint needs a separate object due to the lack of an `id`.